fbw-a380x/src/systems/systems-host/index.ts

```diff
@@ -120,6 +121,8 @@ class SystemsHost extends BaseInstrument {
     this.gpws = new LegacyGpws(this.soundManager);
     this.gpws.init();
 
+    this.backplane.addInstrument('TcasComputer', new LegacyTcasComputer(this.bus, this.soundManager));
```

Comment:
Why is this down here? Could you move it up to the other backplane add calls? I.e. line 117

Reply:
Lowest it can be is line 120 as it needs this.soundManager as a parameter.

Reply:
Those should all be constructed at the same place they're defined TBH.

Reply:
Ah, I see. Overall, I agree that systems-host/index.ts needs a refactor, but I don't think this PR is the right place for it.
https://github.com/flybywiresim/aircraft/issues/9046